serverless/lambda/src/test/java/uk/gov/di/lambdas/UpdateProfileHandlerTest.java

```diff
@@ -77,8 +157,36 @@ public void shouldReturn400WhenRequestIsMissingParameters() {
 
     private void usingValidSession() {
         when(sessionService.getSessionFromRequestHeaders(anyMap()))
+                .thenReturn(
+                        Optional.of(new Session(SESSION_ID).setEmailAddress(TEST_EMAIL_ADDRESS)));
+    }
+
+    private String buildCookieString() {
```

Comment:
This is a duplicate of some existing code, can we pull this out into a helper or something?